soc/arm/nxp_imx/rt/soc_rt11xx.c

```diff
@@ -689,6 +690,11 @@ static int imxrt_init(void)
 void z_arm_platform_init(void)
 {
 	SystemInit();
+
+#if defined(FLEXRAM_RUNTIME_BANKS_USED)
+	/* Configure flexram if not running from RAM */
+	memc_flexram_dt_partition();
+#endif
 }
```

Comment:
Are there any potential issues that may occur with having the FlexRAM be configured after the SystemInit function, considering there will be some push/pop instructions by the time the FlexRAM is reconfigured?